objects/obj_mass_equip/Step_0.gml

```diff
@@ -1,15 +1,17 @@
 
 if (obj_controller.settings=0) or (obj_controller.menu!=23) then instance_destroy();
 
+var romanNumerals;
+romanNumerals= scr_roman_numerals();
+
 if (engage=true){
-    var co,i;co=-1;i=0;
-    
-    repeat(11){co+=1;i=0;
+    for(var co=0, i=0; co<11; co++){
```

Comment:
are you sure this will reset `i` to 0 after each loop pass?

Reply:
it does on line 9, deleted i=0 as its unnecessary now

Reply:
line 9 is just an if. you need to remove the i=0 from line 8